docs/README.md

```diff
@@ -0,0 +1,43 @@
+## Getting started
+
+To get started, do the following:
+
+1. [Request an Echidna token](https://www.w3.org/Web/publications/register) for your spec (W3C Team Members and Chairs only!).
+1. Save the token as a "Secret" in the spec's repository's settings.
+1. Create a ".github/workflows/auto-publish.yml" file at the root of the spec's repository.
+   If you are using VS Code:
+   ```bash"
+   code .github/workflows/auto-publish.yml
+   ```
```

Comment:
I think people know how to create files 😅 
```suggestion
   If you are using VS Code:
   ```bash"
   code .github/workflows/auto-publish.yml
   ```
```